Model/RemotePageRepository.php

```diff
@@ -41,44 +36,63 @@ class RemotePageRepository
      * @var CacheInterface
      */
     private $cache;
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
 
     public function __construct(
         ClientFactory $clientFactory,
-        Configuration $config,
         SiteRepositoryInterface $siteRepository,
         SearchCriteriaBuilder $searchCriteriaBuilder,
         StateInterface $cacheState,
-        CacheInterface $cache
+        CacheInterface $cache,
+        LoggerInterface $logger
     ) {
-        $this->client = $clientFactory->create();
-        $this->config = $config;
+        $this->clientFactory = $clientFactory;
         $this->siteRepository = $siteRepository;
         $this->searchCriteriaBuilder = $searchCriteriaBuilder;
         $this->cacheState = $cacheState;
         $this->cache = $cache;
+        $this->logger = $logger;
     }
 
+    /**
+     * @return array
+     * @throws LocalizedException
+     */
     public function getList(): array
```

Comment:
Je gebruikt strict typing en toch heb je comments toegevoegd, dat hadden we toch niet afgesproken? Of komt dat omdat je wilt noteren dat het een Exception can gooien?

Reply:
Exception!